Jenkinsfile

```diff
@@ -1422,23 +1422,24 @@ def junitAndStore(Map params = [:]){
 def runbld() {
   catchError(buildResult: 'SUCCESS', message: 'runbld post build action failed.') {
     if (stashedTestReports) {
+      def jobName = isPR() ? 'elastic+beats+pull-request' : 'elastic+beats'
+      deleteDir()
+      unstashV2(name: 'source', bucket: "${JOB_GCS_BUCKET}", credentialsId: "${JOB_GCS_CREDENTIALS}")
       dir("${env.BASE_DIR}") {
-        sh(label: 'Prepare workspace context',
```

Comment:
Is this just being replaced by the `deleteDir()` on L1426?

Reply:
Yep, good catch as I didn't explain it. Using the same prepare context as we do normally with L1426-L1427. It was easier than doing the weird find/delete script :)